supervisor/supervisor.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/starkandwayne/shield/timespec"
 )
 
+var debugMessages = true
+
```

Comment:
We're going to use log15 on the API side, so that we don't have global variables hanging out in our packages.